contrib/mpi-proxy-split/p2p_drain_send_recv.cpp

```diff
@@ -135,6 +138,7 @@ completePendingIrecvs()
                                               call->comm);
         g_recvBytesByRank[worldRank] += call->count * size;
         bytesReceived += call->count * size;
+        JALLOC_HELPER_FREE(call);
```

Comment:
I'll have to check, but IIRC, the call object couldn't be freed here because it was being referenced in some other place later. So, I had decided to live with the memory leak. I'll have to go through the code again to make sure.

Reply:
In this case, shall we use a `shared_ptr`